typescript/opensearch/cwlogs_ingestion/resources/lambda/cw_subscription_filter/handler.py

```diff
@@ -0,0 +1,57 @@
+import base64
+import gzip
+import logging
+import json
+import jmespath
+import requests
+import os
+from datetime import datetime
+from requests_auth_aws_sigv4 import AWSSigV4
+
+def cw_subscription_handler(event, context):
+
+    """Extract the data from the event"""
+    data = jmespath.search("awslogs.data", event)
```

Comment:
Is there any limit to the amount of data in each call? OSI only accepts sizes of 10mb or less by default.